index.html

```diff
@@ -20,20 +19,23 @@
       </div>
       <nav class="menu-mobile">
         <ul class="nav-list">
-          <li><a href="#portfolio">Portfolio</a></li>
-          <li><a href="#about">About</a></li>
-          <li><a href="#contact">Contact</a></li>
+          <li class="menuMobile closeBtn">
+            <a href="javascript:void(0)" class="evans">&times;</a>
+          </li>
+          <li><a class="link" href="#portfolio">Portfolio</a></li>
+          <li><a class="link" href="#about">About</a></li>
+          <li><a class="link" href="#contact">Contact</a></li>
         </ul>

```

Comment:
- [ ] Please make sure the mobile menu follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins). Take a look at the below screenshots and work on the differences 👇 
- Figma design 👇 
![Screenshot (225)](https://user-images.githubusercontent.com/67087664/157222352-3f198402-6a25-4e7a-9888-56dade757ec7.png)
- Yours 👇 
![Screenshot (226)](https://user-images.githubusercontent.com/67087664/157222519-ac44e444-1e1c-4480-a4d0-34696d774930.png)

Reply:
Done